pkg/api/routes_scene.go

```diff
@@ -42,13 +42,29 @@ func (rs sceneRoutes) Routes() chi.Router {
 // region Handlers
 
 func (rs sceneRoutes) Stream(w http.ResponseWriter, r *http.Request) {
+
+	//TODO get user agent and use to fine tune media compatibility
+	//ua := r.Header.Get("User-Agent")
+	//logger.Infof("[stream] user agent is: %s \n", ua)
```

Comment:
If we're not intended on using browser detection, then we can remove this.

Reply:
it was left there for internal testing. will be removed once i remove the WIP